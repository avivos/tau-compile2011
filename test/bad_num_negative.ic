int malo_numero = -2147483649;