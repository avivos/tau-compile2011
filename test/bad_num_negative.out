1: INT
1: ID(malo_numero)
1: ASSIGN
1: MINUS
Line 1: Integer out of range: 2147483649
